src/app/components/home/home.component.scss

```diff
@@ -0,0 +1,91 @@
+  * {
+    box-sizing: border-box; 
+  }
+  
+  body {
+    margin: 0; 
+    font-weight: 500;
+    font-family: 'Helvetica Neue';
+  }
+  
+  section {
+    width: 100%;
+    padding: 0 7%;
+    display: table;
+    margin: 0;
+    max-width: none;
+    height: 100vh;
+    background-color: rgba(230, 76, 10, 0.623);
+    font-family: 'Helvetica Neue';
+    
+    &:nth-of-type(2n) {
+      background-color:    rgba(102, 124, 137, 0.705);
+    }
+  }
+  
+  .intro {
+    height: 90vh;
+  }
+  
+  .content {
+    display: table-cell;
+    vertical-align: middle;
+  }
+  
+  h1 {
+    font-size: 3em;
+    display: block;
+    color: white;
+    font-weight: 300;
+  }
+  
+  p {
+    font-size: 1.5em;
+    font-weight: 400;
+    color: white;
+    margin-right: 200px;
+  }
+
+  mat-grid-tile {
+      font-size: 1.5em;
+      font-weight: 500;
+      color: #373B43;
+      margin-top: 0;
+  }
+
+  li {
+    font-size: 1.5em;
+    font-weight: 500;
+    color: #373B43;
+    list-style-type: circle;
+    margin-top: 0;
+}
+  
+  a {
+    font-weight: 700;
+    color: white;
+
+    position: relative;
+    
+    &:hover{ 
+      opacity: 0.8;
+    }
+    
+    &:active {
+      top: 1px;
+    }
+  }
+
+  footer {
```

Comment:
Currently not in use but may be beneficial for a more polished look to our website (not very important though)